src/control/server/storage/scm/provider.go

```diff
@@ -641,6 +653,8 @@ func (p *Provider) formatRamdisk(req FormatRequest) (*FormatResponse, error) {
 	return &FormatResponse{
 		Mountpoint: res.Target,
 		Formatted:  res.Mounted,
+		Mounted:    res.Mounted,
```

Comment:
Not a fan of having the res.Mounted argument be overloaded to imply 3 different states.

Reply:
Yeah per the above response to Joel I'm going to clean this up to be more clear.